packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -514,7 +522,7 @@ class CachedArtifacts implements Artifacts {
         // TODO(jonahwilliams): remove once debug desktop artifacts are uploaded
         // under a separate directory from the host artifacts.
         // https://github.com/flutter/flutter/issues/38935
-        String platformDirName = getNameForTargetPlatform(platform);
+        String platformDirName = _platformDirName(platform);
```

Comment:
I realize you probably named `_platformDirName` off these locals, but you can name `_platformDirName` something more descriptive?  Maybe `_enginePlatformDirectoryName`?

Reply:
Sure. `_enginePlatformDirectoryName` sounds good.